cashctrl_ledger/ledger_utils.py

```diff
@@ -0,0 +1,20 @@
+"""
+This module provides utilities for handling and manipulating pandas DataFrames.
+"""
+
+import pandas as pd
+
+def df_to_str(target: pd.DataFrame) -> str:
```

Comment:
How about adding an argument `index: bool =False`. If True, the index is included in the string representation.

This would make the function more generic and clearly document, that the index is ignored by default.